### ISSUE TITLE
Lists in boutiques task forms are ordered wrong

### ISSUE THREAD

**Issue by MontrealSergiy** [aces/cbrain#830]
A boutiques descriptor can contains a description for an input of type list, which contains several items. E.g.

```json
        {
            "command-line-flag": "--block-window-length", 
            "default-value": [
                10, 
                1, 
                30
            ], 
            "id": "block_window_length", 
            "integer": true, 
            "list": true, 
            "max-list-entries": 3, 
            "min-list-entries": 3, 
            "minimum": 1, 
            "optional": true, 
            "type": "Number" 
        }, 
```

What happens is that when the form is generated, three input fields are shown, but the values are inserted in the wrong order:

![image (1)](https://user-images.githubusercontent.com/11949081/60295455-b738bd00-98f1-11e9-9419-63e5143976ae.png)

Observed the order is distorted (e.g. 30, 10, 1 instead of 10,1,30)

**Comment by MontrealSergiy** [comment]
To reproduce create a boutiques with a list parameter. Set a default value e.g. 10,1,30

Import boutiques to cbrain. 

Create a task corresponding to the boutiques

Expected : the order of values is preserved 
Observed the order is distorted

**Comment by MontrealSergiy** [comment]
The issue was caught on spark boutiques (eeg plugins). Also might reproduce itself on reopening task.

**Comment by MontrealSergiy** [comment]
Snapshot (you can spot the difference between the description of field and values order)
![image (1)](https://user-images.githubusercontent.com/11949081/60284044-957f0c00-98d8-11e9-94fc-590e7f32b64e.png)

**Comment by prioux** [comment]
Fixed in #832
